metadata-ingestion/setup.py

```diff
@@ -258,7 +258,7 @@ def get_long_description():
 
 databricks = {
     # 0.1.11 appears to have authentication issues with azure databricks
-    "databricks-sdk>=0.1.1, != 0.1.11",
+    "databricks-sdk>=0.9.0",
```

Comment:
There's some minor API changes that I don't want to be backwards-compatible to. They're pretty consistently updating this library and I'd like to keep up with it